CODEOWNERS

```diff
@@ -0,0 +1,53 @@
+# Remember that the last applicable rule for any given file is the only one
+# that applies (if I am reading the docs correctly).
+
+# Default rule: @diyessi gets everything.
+*					@diyessi
+
+/cmake/					@rkimballn1 @silee2
+
+/.ci/					@crlishka
+/contrib/docker/			@crlishka
+/.travis.yml				@crlishka @postrational
```

Comment:
Let's give /.travis.yml to @postrational only.  I know almost nothing about how Travis CI is running, as it was developed by Michal.

Reply:
Done